server/test/estree-utils-test.ts

```diff
@@ -1,4 +1,4 @@
-import { expect } from 'chai';
+const { expect } = require('chai');
```

Comment:
Maybe we should include the typings then? https://www.npmjs.com/package/@types/chai